Makefile

```diff
@@ -45,6 +45,11 @@ devtools:
 
 suavedevtools:
 	go run ./suave/gen/main.go -write
+	cd core/types && \
```

Comment:
Could you use // go:generate to do this? https://github.com/flashbots/suave-execution-geth/blob/feature/e2e-block-building-builderapi/suave/builder/api/api.go#L15